AutoRest/TestServer/server/routes/azureParameterGrouping.js

```diff
@@ -0,0 +1,50 @@
+var express = require('express');
```

Comment:
@yugangw-msft None of these testserver .js files seem to have the license header